clients/web/src/views/layout/FooterView.js

```diff
@@ -9,9 +9,18 @@ import 'girder/stylesheets/layout/footer.styl';
  * This view shows the footer in the layout.
  */
 var LayoutFooterView = View.extend({
+
+    initialize: function (settings) {
+        const mail = settings.contactEmail || '';
```

Comment:
In `initialize`, I think we should always set `this.contactEmail` to be equal to `settings.contactEmail`, with a default of `null` if it's not provided.

Then, in `render`, we should conditionally apply the concatenation with `mailto:` (encoding it if necessary), then pass that result (or `null`) into the template.

This will also `LayoutFooterView` to be subclassed such that it takes the same parameters, but renders them in a different way.